internal/campaigns/graphql/repository.go

```diff
@@ -81,6 +81,14 @@ func (r *Repository) Slug() string {
 	return strings.ReplaceAll(r.Name, "/", "-") + "-" + r.Rev()
 }
 
+func (r *Repository) SlugForPath(path string) string {
+	name := r.Name
+	if path != "" {
+		name = name + "-" + path
+	}
+	return strings.ReplaceAll(name, "/", "-") + "-" + r.Rev()
```

Comment:
Since this is a one way transformation, we might want to hash instead here to avoid any issues with other characters that are disallowed in paths on more restrictive platforms like Windows.

Reply:
Done! I decided to only hash the `path` so that the temp files will be somewhat human-readable.